### ISSUE TITLE
Support flexible workflows and outputs

### ISSUE THREAD

**Issue by ericcz0** [c4pt0r/tcli#11]
I would like to propose some changes that help support more flexible workflows; namely, piping formatted key-value operation results to output files:

- [x] Redirect non-result console output to Stderr so that Stdout only receives results of key-value operations
- [ ] Implement options to configure formatting of key-value output (i.e., JSON, csv)

**Comment by c4pt0r** [comment]
Thank you for your contribution, already merged.

**Comment by ericcz0** [comment]
Would it be worth implementing to change or allow configuration of the way JSON is output? The fix in #14 uses the JSON convention used by the existing implementation:
```
[
    {
        "K": "key",
        "V": "value"
    }
]
```
but there may be other ways to output to JSON. We could represent each key-value pair as a single field in a JSON:
```
[
    {
        "key": "value"
    }
]
```
Using this convention, we can also output all results to a single JSON object rather than a list of objects:
```
{
    "key1": "value1",
    "key2": "value2",
    ...
}

```
